### DISCUSSION TITLE
unified js website isn't showing its favicon

### DISCUSSION THREAD

**Discussion by yadunandanbhat** [unifiedjs/unified#189] [General] [answered]
See below image
![image](https://user-images.githubusercontent.com/54680617/168088240-bb49f07c-f674-45d1-990e-e072ae42f7fa.png)
And no, its not my internet or browser's problem.

**Comment by ChristianMurphy** [comment, marked as answer]
The unified site isn't showing a favicon, because it currently doesn't have one.
You're welcome to open a PR converting to logo to a favicon and adding it to the site.

**Reply by yadunandanbhat** [reply]
Will try!
